src/components/SharedComponents/Accordion.tsx

```diff
@@ -1,20 +1,24 @@
 import React from 'react';
-import { AccordionDetails } from '@material-ui/core';
+import { AccordionDetails, AccordionProps } from '@material-ui/core';
 import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import {
   StyledMuiAccordion,
   StyledAccordionSummary,
   MarkdownBody,
 } from './Accordion.style';
 
-const StyledAccordion = (props: {
+export interface StyledAccordionProps extends Omit<AccordionProps, 'children'> {
```

Comment:
does this update syntax serve a specific purpose in this case?

Reply:
yes! It allows us to pass the `expanded` and `onChange` props to the `StyledAccordion`, as well as any other `MuiAccordion` props